### PR TITLE
Create chapter_two - texify.txt

### PR DESCRIPTION
**Pull request by alvinjchitena**
I created chapter_two - texify.txt. It's the text version of the tex for chapter two.

@mkphuthi I like your texify rules. I added some comments in the committed file for some changes, additions & clarifications needed in the rules to make sure that we can work independently with minimal questions and mistakes.

I will list the comments out of context below, for ease of access:

Issues raised:

Handling main chapters e.g. Chapter 2 Variables
Sub-chapters vs headings e.g. 2.1 Variables vs Variables
Formula box for handling stuff like [speed = distance x time]
Where does metadata about the specific Fig. go in your Fig. template
What do do with a) b) c) d) ...
Specific handling for toolboxes?

I suggest this structure in your texify rules:
- Main Chapter
- Sub-Chapter
- Headings
- Exercises
- Examples
- Demonstrations
- Script
- Shell
- Numbered
- Bullet listed
- Letter numbered
- Figs.
- Bold
- Italics
- Underline
- Caps

Please also specify if your tex is space/line sensitive. E.g should \items in a numbered list be line spaced or not, or it doesn't matter. This will reduce editing (if any) on your end.

### CODE REVIEW

File: 12-week Course/Textbook Source/Chapter_Two/chapter_two - texify.txt [hunk 1]
```diff
@@ -0,0 +1,187 @@
+Chapter 2 – Variables
```
Comment (alvinjchitena) [inline 12-week Course/Textbook Source/Chapter_Two/chapter_two - texify.txt]: Should we worry about the Main Chapters? Perhaps add something to handle them as well?

File: 12-week Course/Textbook Source/Chapter_Two/chapter_two - texify.txt [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+\section{Abstraction}
```
Comment (alvinjchitena) [inline 12-week Course/Textbook Source/Chapter_Two/chapter_two - texify.txt]: Do these apply equally to sub-chapters AND headings? e.g:
2.1 Variables AND Variables

File: 12-week Course/Textbook Source/Chapter_Two/chapter_two - texify.txt [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+
+So far, we have only been considering things that an ordinary calculator can do with just a few simple
+operations. The first and probably the most powerful tool in programming is abstraction. Abstraction
+is the use of an idea rather than an event. This is best shown through examples.
+
+\begin{example} \textbf{Example 2.1 – Using a formula}
+If I were to tell you that to calculate the distance travelled by a car I did the following calculation:
+
+distance = 2 * 3 = 6
```
Comment (alvinjchitena) [inline 12-week Course/Textbook Source/Chapter_Two/chapter_two - texify.txt]: Do we need a formula box? I.e something that holds things that aren't scripts, shells, examples of lists etc? In the og book, these were in boxes.

File: 12-week Course/Textbook Source/Chapter_Two/chapter_two - texify.txt [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+abstraction. Rather than using an event (i.e. exact numbers) we use an idea (i.e. a formula).
+\end{example}
+
+\begin{exercise} \textbf{Supplementary Exercise 2.1 (Science) – Making a complicated calculation easy}
+Imagine that you are the assistant to a crazy physicist named Larmour who needs to know the power
+radiated from a point (The point is that this is a complicated calculation, you do not even need to
+know what it means). Larmour says he wrote his formula on the board and needs the answer to
+finish making a machine that gets rid of all homework so you happily help him. The formula is:
+
+(Figure 1) Formula {a}
```
Comment (alvinjchitena) [inline 12-week Course/Textbook Source/Chapter_Two/chapter_two - texify.txt]: Wasn't sure were to put the general description of the specific Fig. inside your tex. Can you specify what to do with these?

File: 12-week Course/Textbook Source/Chapter_Two/chapter_two - texify.txt [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+\item Each person can have a maximum of one number.
+\item Whenever the interpreter says something like “print(x+y)”, the calculator checks the person
+called x and checks the person called y and shouts the answer.
+\end{enumerate}
+
+Shout at a few instructions declaring people as variables, printing variables, exchanging variables and
+adding/subtracting variables and assigning them to new ones etc. Have fun!
+
+\begin{exercise} \textbf{Exercise 2.2: Python Siri}
+a) Now that we can code, we want to make our own version of iPhone’s Siri. Write a script that asks
```
Comment (alvinjchitena) [inline 12-week Course/Textbook Source/Chapter_Two/chapter_two - texify.txt]: This is neither "numbered" nor "itemized." How do we handle these?

File: 12-week Course/Textbook Source/Chapter_Two/chapter_two - texify.txt [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+%\addcontentsline{toc}{figure}{Figure \ref{fig:placeholder}} % Uncomment to add the figure to the table of contents
+\end{figure}
+
+Another type of error is what is called a semantic error, this is an error where your code runs but it
+does not do what you want it to, for example if you want to add two numbers but Python subtracts
+them instead. These errors are very dangerous because Python will not tell you that your code is
+wrong, so you have to test your code and if it has problems, you “debug” it. Debugging means fixing
+code that does not work properly.
+
+\section{Toolbox 2.1}
```
Comment (alvinjchitena) [inline 12-week Course/Textbook Source/Chapter_Two/chapter_two - texify.txt]: Is there a specific handling for Toolboxes?

**Comment by mkphuthi** [comment]
Thanks for bringing all of these up! I think I have addressed all your questions in the README. For many of the cases where the actual latex is simple such as \section{} compared to  changing it to \subchapter{}, I will leave as is to make compiling and debugging easier and faster.

**Pull request merged**